README.md

```diff
@@ -1,12 +1,12 @@
 # Suomi.fi Design System
 
-Documentation website for the [Suomi.fi Design System](https://vrk-kpa.github.io/suomifi-design-system/fi/)
+Documentation website for the [Suomi.fi Design System](https://vrk-kpa.github.io/suomifi-design-system/)
 
```

Comment:
https://vrk-kpa.github.io/suomifi-design-system/ is mentioned three times. Maybe 1 would be enough?

Reply:
Yes, that is a good point. We could rewrite this README on that part.

Reply:
README will be rewritten at better time.